packages/devtools_extensions/example/foo/.vscode/launch.json

```diff
@@ -0,0 +1,57 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "foo_devtools_extension",
```

Comment:
Is there much value in running without the simulated environment? If not, maybe we should remove this or put the simulated environment at the top (which is basically the default if it's the first time you open the example).